modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -854,6 +854,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-175201 -Verify if is possible create a validation using the oldValue function with the Text field using Commerce Order object"
+	@priority = 5
+	test CanCreateValidationUsingOldValueFunctionInSystemObject {
+		task ("Given: an account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Accounts Name",
+				accountEntryType = "Business");
+		}
+
+		task ("And given: a custom validation using the oldValue function is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Postal Address");
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.addObjectValidation(
+				validationName = "Custom Validation",
+				validationType = "Expression Builder");
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+			ObjectCustomValidation.gotoTab(tabName = "Conditions");
+
+			ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "oldValue('Name') == 'Test'");
+
+			ObjectCustomValidation.addErrorMessage(errorMessage = "Invalid Value");
+
+			PortletEntry.save();
+		}
+
+		task ("And given: an address is created to the account") {
+			Account.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Accounts Name");
+
+			Account.gotoAssigneeTab(assigneeTab = "Addresses");
+
+			Account.addAddress(
+				addressCity = "Diamond Bar",
+				addressCountry = "United States",
+				addressName = "Name Test",
+				addressPostalCode = 91765,
+				addressRegion = "California",
+				addressStreet1 = "1400 Montefino Ave",
+				addressType = "Billing");
+		}
+
+		task ("When: the validation is activated") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Postal Address");
+
+			ObjectAdmin.goToValidationsTab();
+
+			ObjectCustomValidation.gotoCustomValidation(validationName = "Custom Validation");
+
+			ObjectCustomValidation.activeValidation();
+
+			PortletEntry.save();
+		}
+
+		task ("And when: the address from created account is edited") {
+			Account.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Accounts Name");
+
+			Account.gotoAssigneeTab(assigneeTab = "Addresses");
+
+			LexiconEntry.gotoEntry(rowEntry = "Name Test");
```

Comment:
This must be something like: `LexiconEntry.gotoEntry(rowEntry = "Test");` because the name of the address will be the same on the first one